### PR TITLE
Mobile FAQ updates

### PR DESCRIPTION
**Pull request by lauramthomas**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by sweeneyskirt-sl** [comment]
Hi @lauramthomas and @skhg -- just checking in on the status of this PR. Is it ready to go? @skhg -- looks like we might be waiting on approval from you?

**Comment by lauramthomas** [comment]
We have some further work to do, so the PR can stay open.

File: docs/mobile-apps/faq.md [hunk 1]
```diff
@@ -56,6 +56,13 @@ Yes. See [Camera Image Injection](/mobile-apps/features/camera-image-injection).
 * **Android emulators** - not supported.
 * **iOS simulators** - not supported.
 
+#### **Is there a maximum time I can use a real device for either manual or automated testing, provided I actually do interact with the device?**
+
+For real devices on the public cloud, the limit is 6 hours; for private cloud devices there is no limit.
+
+#### **Can I upload files to or download files from a real device (iOS/Android)?
```
Comment (sweeneyskirt-sl) [inline docs/mobile-apps/faq.md:63]: Hey @lauramthomas -- This heading is missing the `**` at the end to close the "bold" tag, but also, you don't need to add bold to these headings. Our H4 CSS definition already makes these headings bold, so these will look identical with or without the `**`.

File: docs/mobile-apps/faq.md [hunk 2]
```diff
@@ -56,6 +56,13 @@ Yes. See [Camera Image Injection](/mobile-apps/features/camera-image-injection).
...
+
+Yes, by using virtual USB on a private device. See [Virtual USB Testing on Real Mobile Devices](https://docs.saucelabs.com/mobile-apps/virtual-usb) for more information.
```
[the remark this answers is not in the document]
Comment (lauramthomas) [reply docs/mobile-apps/faq.md:65]: @skhg Do you know if this is correct?

[the remark this answers is not in the document]
Comment (wswebcreation) [reply docs/mobile-apps/faq.md:65]: There are multiple buts/options here:

- vUSB can only work on private devices and you can only push files to Android devices with vUSB, you can't with iOS
- in live testing you can upload files to android devices, there is no way to download them
- iOS does not have an option to upload/download files from the devices on real devices
- Appium can push files to android emulators and real devices
- Appium can download files from Android emulators and real devices
- Appium can push files to iOS simulators, you can't pull

So I wonder what we need to tell here

[the remark this answers is not in the document]
Comment (lauramthomas) [reply docs/mobile-apps/faq.md:65]: @wswebcreation So, is "Can I upload files to or download files from a real device (iOS/Android)?/Yes, by using virtual USB on a private device." correct?

[the remark this answers is not in the document]
Comment (wswebcreation) [reply docs/mobile-apps/faq.md:65]: yes, but only with Android

[the remark this answers is not in the document]
Comment (lauramthomas) [reply docs/mobile-apps/faq.md:65]: @wswebcreation Got it!

**Pull request merged**